lib/taiko.js

```diff
@@ -1767,12 +1768,19 @@ module.exports.inputField = (attrValuePairs, ...args) => {
  */
 module.exports.fileField = fileField;
 
-function fileField(attrValuePairs, ...args) {
+function fileField(attrValuePairs, options = {}, ...args) {
     validate();
+    if( options instanceof RelativeSearchElement) {
```

Comment:
this check can be moved inside `getValues` which is used in common for most selectors